src/WCSimConstructGeometryTables.cc

```diff
@@ -69,6 +69,7 @@ void WCSimDetectorConstruction::GetWCGeom
     }
 
     if ((aPV->GetName() == "WCCapBlackSheet") || (aPV->GetName() == "glassFaceWCPMT")){
```

Comment:
I'm getting an error when I compile this related to an extra {. I think it's because you meant to remove this if statement and replace it with the one below.